packages/liveblocks-core/src/__tests__/_utils.ts

```diff
@@ -451,14 +451,14 @@ export function prepareStorageUpdateTest<
 ): () => Promise<void> {
   return async () => {
     const { storage: refStorage, machine: refMachine } =
-      await prepareRoomWithStorage(items, 1);
+      await prepareRoomWithStorage(items, -1);
 
     const { storage, machine } = await prepareRoomWithStorage<
       TPresence,
       TStorage,
       TUserMeta,
       TRoomEvent
-    >(items, 0, (messages) => {
+    >(items, 1, (messages) => {
```

Comment:
Perhaps use `-2` here, so that even if we write tests with more than one actor, we'll not run into the same problem in the future?